applications_tests/gls_sharp_navier_stokes/pp_contact_test.prm

```diff
@@ -260,6 +260,7 @@ subsection linear solver
     set max iters                                 = 1000
     set relative residual                         = 1e-3
     set minimum residual                          = 1e-11
+    set preconditioner                            = ilu
     set ilu preconditioner fill                   = 0
     set ilu preconditioner absolute tolerance     = 1e-20
     set ilu preconditioner relative tolerance     = 1.00
```

Comment:
(Unrelated to this PR) 
Similar to a previous comment, here (lines 267-274) why are ``amg`` parameters defined when the ``ilu`` preconditioner is used? They are default values except for the ``amg aggregation threshold``.